unit_tests/test_tlslite_messages.py

```diff
@@ -2454,5 +2455,72 @@ def test_write(self):
             b'\xbc\xaa'))
 
 
+class TestHeartbeat(unittest.TestCase):
+    def test___init____(self):
+        heartbeat = Heartbeat()
+
+        self.assertEqual(heartbeat.message_type, 0)
+        self.assertEqual(heartbeat.payload, bytearray(0))
+
+    def test_messageType(self):
+        heartbeat = Heartbeat().create(HeartbeatMessageType.heartbeat_request,
+                                       bytearray(1), 20)
+
+        self.assertEqual("heartbeat_request", heartbeat.messageType)
+
+    def test_wrong_messageType(self):
+        heartbeat = Heartbeat().create(5, bytearray(1), 20)
+
+        self.assertEqual("unknown(5)", heartbeat.messageType)
+
+    def test___str___(self):
```

Comment:
extra `_` at the end :)

the name format is `test_` + name of function. Since function name is `__str__` the name of test should be `test___str__`